server/src/engineModel/EngineModel.ts

```diff
@@ -92,6 +93,15 @@ export class EngineModel {
                 this.completionProvider.addCompletionItem(method);
             });
         }
+
+        if (!!yclass.instances && !!yclass.instances[0].item) {
+            yclass.instances[0].item.forEach((_instance) => {
+                if (ymlClass.extends.indexOf('yseop.lang.Enum') >= 0) {
```

Comment:
# Style

I see people doing “≠ -1” more often than “≥ 0”. There’s this notion of “index of, not found ⇒ -1” in nearly every Java / JS dev’s mind, I think. But I might be wrong. But the `≥` makes me want to double-check that you used 0 and not something else (like you did for `::` in another PR because you wanted to have at least a non-colon character before the `::`). With ≠ there’s less ambiguity as to what you’re trying to do: it’s kinda clear that the actual position doesn’t matter.

LOL the more trifling the matter is, the longer my comments get.

Reply:
I'll use `!= -1`. Thanks.